python/setup.py

```diff
@@ -23,7 +23,11 @@
 
 
 def config_cython():
-    """Try to configure cython and retyurn cython configuration"""
+    """Try to configure cython and return cython configuration"""
+    if os.name == 'nt':
+        print("WARNING: Cython is not supported on Windows, will compile without cython module")
+        return []
```

Comment:
You mean try setp(...) except?

Ok. Let's merge this to make sure build pass then verify that solution works on windows